README.md

```diff
@@ -20,14 +20,17 @@ configurations / clusters.
 Add the following to your `WORKSPACE` file to add the necessary external dependencies:
 
 ```python
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-git_repository(
+# Download the rules_docker repository at release v0.9.0. See
+# https://github.com/bazelbuild/rules_docker#setup for instructions to download
+# the latest available rules_docker release.
+http_archive(
     name = "io_bazel_rules_docker",
-    commit = "{HEAD}",
-    remote = "https://github.com/bazelbuild/rules_docker.git",
+    sha256 = "e513c0ac6534810eb7a14bf025a0f159726753f97f74ab7863c650d26e01d677",
```

Comment:
leave this as a place holder so people are forced to go to rules_docker to get this # and we only have to maintain it there (same for the 0.9.0's)

Reply:
Done